catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -114,15 +120,11 @@ public GeoNamesCatalogIndexer(
     filter =
         filterBuilder.allOf(
             filterBuilder
-                .attribute(Core.METACARD_TAGS)
+                .attribute(METACARD_TAGS)
```

Comment:
✏️ I prefer `Core.METACARD_TAGS` rather than `METACARD_TAGS` because it allows the variable to keep its context which enhances readability. 

I've seen this in a few places where there is a constant like `Core.CLASSIFICATION` and also a `SomeOtherConstantsFile.CLASSIFICATION`

Is there a standard practice for DDF? thoughts?

Reply:
There is no DDF convention that I am aware of but I made the change. It looks like we qualify almost all our uses in DDF.